### ISSUE TITLE
List of things we need to do before we can user test

### ISSUE THREAD

**Issue by davidascher** [participedia/frontend#211]
- [x] make editing of cases, methods and orgs work (some bugs may remain) 
  - [x] frontend work: @davidascher
  - [x] backend work: @dethe
- [x] make videos show up: #202 @samjiangweb and @andreadelrio 
- [x] make the mobile view look OK: #197 @samjiangweb and @andreadelrio 
- [x] make search roughly work:
  - [x] query handling in the API endpoint: https://github.com/participedia/api/issues/53 @dethe 
  - [x] figure out UX of main "all" view: @samjiangweb 
- [x] make profile views work "enough": @davidascher
- [x] make sharing widgets work: #32 @andreadelrio 
- [ ] cross-browser check on IE: #19 ???
- [x] make "featuring" work
  - [x] front-end #214 @davidascher
  - [x] back-end @dethe
  - [x] make sure results show up in home page @dethe 
- [ ] update staging database from v2 site nightly

Not required for first user testing:
- localization flows (both of fixed strings and body content
- history views, undo

(feel free to edit this issue with more requirements and progress)

**Comment by dethe** [comment]
Calling this one done, database updates from V2 can be its own issue.
